### DISCUSSION TITLE
Student cannot register via GitHub Classroom

### DISCUSSION THREAD

**Discussion by foleyj2** [community/community#101550] [GitHub Education]
### Hi everyone,

We are using Github Classroom in a project-based class.  A student with an existing GitHub account has been trying to register with her identifier to the classroom.  Each time she does so, it claims that her repository access has been removed and to contact the instructor (me).  When I look at her account in Classroom, it says she has account @ghost which is clearly not right.  We have tried different computers, a newly created GitHub account, incognito mode, but it keeps doign this error.

**Comment by leomotors** [comment]
Hi, I also found the same problem. Your student's account might be shadow-banned.

You can verify that by visiting their GitHub account and it should be 404 not found. If the account is actually shadow-banned you may need to ask them to contact support themself.
 
Source: https://webapps.stackexchange.com/questions/172282/is-github-shadowbanning-my-account-by-returning-404-to-other-users-who-access-my
